crates/wasi/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "wasmi_wasi"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+wasi-common = "2.0.1"
+wasi-cap-std-sync = "2.0.1"
+wiggle = "2.0.0"
+wasmi = { version = "0.20.0", path = "../wasmi" }
+wasmi_core = { version = "0.5.0", path = "../core"}
```

Comment:
I think it is better if you do not directly depend on `wasmi_core` but use `wasmi_core` through the `wasmi` crate that re-exports `wasmi_core` as `wasmi::core`.

Reply:
Okay. Will fix this